peps/pep-0639.rst

```diff
@@ -524,104 +520,62 @@ paths in the project source tree relative to ``pyproject.toml`` to file(s)
 containing licenses and other legal notices to be distributed with the package.
 It corresponds to the ``License-File`` fields in the Core Metadata.
 
-Its value is a table, which if present MUST contain one of two optional,
-mutually exclusive subkeys, ``paths`` and ``globs``; if both are specified,
-tools MUST raise an error. Both are arrays of strings; the ``paths`` subkey
-contains verbatim file paths, and the ``globs`` subkey valid glob patterns,
-which MUST be parsable by the ``glob`` `module <globmodule_>`__ in the
-Python standard library.
-
+Its value is an array of strings which MUST contain valid glob patterns,
+as specified below.
+The glob patterns MAY contain special glob characters: ``*``, ``?``, ``**``
+and character ranges: ``[]``, and tools MUST support them.
 Path delimiters MUST be the forward slash character (``/``),
 and parent directory indicators (``..``) MUST NOT be used.
 Tools MUST assume that license file content is valid UTF-8 encoded text,
 and SHOULD validate this and raise an error if it is not.
 
-If the ``paths`` subkey is a non-empty array, build tools:
-
-- MUST treat each value as a verbatim, literal file path, and
-  MUST NOT treat them as glob patterns.
-
-- MUST include each listed file in all distribution archives.
-
-- MUST NOT match any additional license files beyond those explicitly
-  statically specified by the user under the ``paths`` subkey.
+Literal paths (e.g. ``LICENSE``) are treated as valid globs which means they
+can also be defined.
 
-- MUST list each file path under a ``License-File`` field in the Core Metadata.
+To achieve better portability, the filenames to match should only contain
+the alphanumeric characters, underscores (``_``), hyphens (``-``)
+and dots (``.`` - for extensions).
```

Comment:
I got confused here because I assumed extension moduels first. This probably means stuff like `.txt` or `.rst`.  OTOH I can have `my.license.file` here as well and I think saying "for extensions" brings no clarity to this rule. I suggest removing that note:

```suggestion
and dots (``.``).
```